frontend/src/styles/_clients.scss

```diff
@@ -0,0 +1,36 @@
+.clients {
+    padding-top: 100px;
+    padding-bottom: 100px;
+}
+
+.clients-list {
+    display: flex;
+    flex-flow: row wrap;
+    margin-left: calc(-1 * var(--card-set-gap));
+    margin-top: calc(-1 * var(--card-set-gap));
+}
+
+.clients-item {
```

Comment:
I suppose item should be included inside of list block.
See the BEM documentation :)